react/AddressForm.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Address } from 'vtex.checkout-graphql'
+import { Input } from 'vtex.styleguide'
+import { useAddressContext } from 'vtex.address-context/AddressContext'
+
+export interface LineFragment {
```

Comment:
This is also defined in the PlaceDetails component. It doesn't seem clean to me to import it from there, can I have a separated file for types/interfaces that are shared among these components? How should I name it? I was thinking about `react/types.tsx`.

Reply:
I really like this idea. I was talking with breno about that and he seems to be into this too.

Reply:
I talked to Fernando too about this, I will be using the `typings` folder with the `d.ts` extension.